pkg/operator/controller/prometheus.go

```diff
@@ -239,7 +239,7 @@ func getAlertRules() []promv1.Rule {
 			"kubevirt_cdi_dataimportcron_outdated_total > 0",
 			"15m",
 			map[string]string{
-				"summary":     "DataImportCron latest imports are outdated",
+				"summary":     "DataImportCron, which is in charge of recurring polling of latest version disk images as PVCs (also known as golden images), is failing to keep images up-to-date",
```

Comment:
Maybe something like `DataImportCron, VM templates disk image sources are not being updated on its defined schedule`
I am having a hard time formulating a good message. I don't like the one we have here.

Reply:
I changed it to a mixture of both the original and your suggestion, let me know if that sounds better